packages/api-queries/src/me-monetize.ts

```diff
@@ -0,0 +1,78 @@
+import {
+	fetchMonetizeSubscriptions,
+	MonetizeSubscriptionAutoRenewResponse,
+	MonetizeSubscriptionStopResponse,
+	requestAutoRenewDisable,
+	requestAutoRenewResume,
+	requestSubscriptionStop,
+} from '@automattic/api-core';
+import { mutationOptions, queryOptions } from '@tanstack/react-query';
+import { queryClient } from './query-client';
+
+export const monetizeSubscriptionsQuery = () =>
+	queryOptions( {
+		queryKey: [ 'me', 'monetize', 'subscriptions' ],
+		queryFn: () => fetchMonetizeSubscriptions(),
+	} );
+
+export const monetizeSubscriptionQuery = ( subscriptionId: string ) =>
+	queryOptions( {
+		...monetizeSubscriptionsQuery(),
+		enabled: !! subscriptionId,
```

Comment:
We now have an "official" guideline for this:

https://github.com/Automattic/wp-calypso/blob/054e228306e2a907b9caed7b3b2cdf4460ce17f6/packages/api-queries/README.md?plain=1#L49

We should let the component to add `enabled` when necessary. Also currently, with this inline logic here, it would seems that `subscriptionId` is sometimes unavailable, but the type is `string` (not `?: string`). It's better to assume that this function will always accept a defined `subscriptionId`, and let the component itself pass `enabled` when not.